ecs.go

```diff
@@ -118,20 +118,28 @@ func Drain(ecsCluster, ec2Instance string) error {
 }
 
 func getContainerInstance(ecsCluster, ec2Instance string) (*ecs.ContainerInstance, error) {
-	respList, err := ecsClient.ListContainerInstances(&ecs.ListContainerInstancesInput{Cluster: &ecsCluster})
-	if err != nil {
-		return nil, err
-	}
+	var containerInstances []*ecs.ContainerInstance
+
+	err := ecsClient.ListContainerInstancesPages(&ecs.ListContainerInstancesInput{Cluster: &ecsCluster}, func(page *ecs.ListContainerInstancesOutput, lastPage bool) bool {
+		respInstances, err := ecsClient.DescribeContainerInstances(&ecs.DescribeContainerInstancesInput{
+			Cluster:            &ecsCluster,
+			ContainerInstances: page.ContainerInstanceArns,
+		})
 
-	respInstances, err := ecsClient.DescribeContainerInstances(&ecs.DescribeContainerInstancesInput{
-		Cluster:            &ecsCluster,
-		ContainerInstances: respList.ContainerInstanceArns,
+		if err != nil {
+			fmt.Printf("Error describing container instances: %s", err)
+			return false
+		}
+
+		containerInstances = append(containerInstances, respInstances.ContainerInstances...)
+		return true
```

Comment:
I think it would be better to use here something like this:

```go
return !lastPage
```

Just to make sure we don't call an iterator in a loop for no reason?